miden-tx/src/tests/mod.rs

```diff
@@ -106,13 +101,10 @@ fn transaction_executor_witness() {
 
 #[test]
 fn executed_transaction_account_delta() {
-    let tx_context = TransactionContextBuilder::with_standard_account(ONE)
+    let mut tx_context = TransactionContextBuilder::with_standard_account(ONE)
         .with_mock_notes_preserved_with_account_vault_delta()
```

Comment:
Not for this PR, but I find methods like this a bit confusing. I wonder if there is a better way to approach this.